lib/MetaCPAN/Document/File.pm

```diff
@@ -32,6 +32,20 @@ sub BUILD {
 
 =head1 PROPERTIES
 
+=head2 deprecated
+
+Indicates file deprecation (this will normally mean the abstract will contain
```

Comment:
Will the file get the 'deprecated' flag if the entire release is deprecated, as well as the individual file is deprecated? If so, the documentation isn't quite right: it would be something more like:

    Indicates file deprecation (the abstract contains "DEPRECATED" or "DEPRECIATED", or the x_deprecated flag is included in the corresponding "provides" entry in distribution metadata); it is also set if the entire release is marked deprecated (see L<MetaCPAN::Document::Release#deprecated>).

Reply:
added as suggested